pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -406,6 +453,15 @@ var _ = Describe("Converter", func() {
 	})
 })
 
+func diskToDiskXML(disk *v1.Disk) string {
+	devicePerBus := make(map[string]int)
+	libvirtDisk := &Disk{}
+	Expect(Convert_v1_Disk_To_api_Disk(disk, libvirtDisk, devicePerBus)).To(Succeed())
+	data, err := xml.MarshalIndent(libvirtDisk, "", "  ")
+	Expect(err).ToNot(HaveOccurred())
+	return strings.ToLower(string(data))
```

Comment:
Why is there a `ToLower`?

Reply:
The XML marshaler, per its documentation, only works on and returns capitalized root objects. Libvirt would handle it as-is, I could instead make the expected XML capitalized.